onvm/onvm_mgr/onvm_stats.c

```diff
@@ -566,7 +566,7 @@ onvm_stats_display_nfs(unsigned difftime, uint8_t verbosity_level) {
                 }
         }
 
-        if (ONVM_ENABLE_SHARED_CPU) {
+        if (ONVM_NF_CORE_SHARING) {
```

Comment:
Can we change the next line to `fprintf(stats_out, "\n\nShared Core stats\n");` while we're here?